itou/templates/job_seekers_views/details.html

```diff
@@ -0,0 +1,111 @@
+                                    </div>
+                                    {% if iae_eligibility_diagnosis %}
+                                        {% include "job_seekers_views/includes/eligibility_diagnosis.html" with eligibility_diagnosis=iae_eligibility_diagnosis kind="IAE" only %}
+                                    {% endif %}
+                                    {% if geiq_eligibility_diagnosis %}
+                                        {% include "job_seekers_views/includes/eligibility_diagnosis.html" with eligibility_diagnosis=geiq_eligibility_diagnosis kind="GEIQ" with_allowance=request.user.is_employer only %}
+                                    {% endif %}
+                                </div>
+                                {% if approval %}
+                                    <div class="col-12 col-lg-4 order-1 order-lg-2 mt-lg-6">
+                                        {% include "approvals/includes/box.html" with approval=approval only %}
+                                    </div>
+                                {% endif %}
+                            </div>
+                        </div>
+                        <div class="tab-pane fade" id="candidatures" role="tabpanel" aria-labelledby="job-applications-tab">
+                            <h2>Candidatures envoyées</h2>
+                            <hr>
+                            {% for job_application in sent_job_applications %}
+                                <div class="c-box c-box--results has-links-inside my-3 my-md-4">
+                                    <div class="c-box--results__header">
+                                        {% include "approvals/includes/job_description_creation.html" with job_application=job_application %}
+                                    </div>
+                                    <hr class="m-0">
+                                    <div class="c-box--results__body">
+                                        <div class="c-box--results__list-data">
+                                            {% include "approvals/includes/job_description_list.html" with job_application=job_application %}
+                                        </div>
+                                    </div>
+                                    <div class="c-box--results__footer">
+                                        <div class="d-flex justify-content-end">
+                                            <a href="{% url 'apply:details_for_prescriber' job_application_id=job_application.id %}?back_url={{ request.get_full_path|urlencode }}"
+                                               class="btn btn-outline-primary btn-block w-100 w-md-auto"
+                                               {% matomo_event "candidat" "clic" "detail-candidature" %}
+                                               aria-label="Voir la candidature de {{ job_application.sender.get_full_name }}">Voir sa candidature</a>
```

Comment:
C'est un peu vicieux car je pense que pour les pro on s'en fiche (ou peut-être pas ?) mais je pense au cas où c'est le candidat qui à envoyé la candidature.
```suggestion
                                               aria-label="Voir la candidature de {{ job_application.sender.get_full_name|mask_unless:can_view_personal_information }}">Voir sa candidature</a>
```

Reply:
Je me le note pour une autre PR.